test/modules/TwinTester/Program.cs

```diff
@@ -22,15 +22,14 @@ static async Task Main()
 
             try
             {
-                await Task.Delay(Settings.Current.TestStartDelay);
-
                 RegistryManager registryManager = RegistryManager.CreateFromConnectionString(Settings.Current.ServiceClientConnectionString);
 
-                using (ITwinTestInitializer twinOperator = await GetTwinOperatorAsync(registryManager))
-                {
-                    await twinOperator.Start();
-                    await Task.Delay(Settings.Current.TestDuration, cts.Token);
-                }
+                ITwinTestInitializer twinOperator = await GetTwinOperatorAsync(registryManager);
```

Comment:
does registryManager implement IDisposable, right? if so, please call Dispose in Try...Finally.